db/migrate/20190202153502_create_bikes.rb

```diff
@@ -0,0 +1,17 @@
+class CreateBikes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :bikes do |t|
+      t.integer :size
+      t.string :size_unit
+      t.string :color
+      t.string :bike_type
```

Comment:
I thought we were going with 'discipline'.

Reply:
Updated